CrewChiefV4.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.40629.0
+# Visual Studio 15
```

Comment:
please make sure solution is buildable and can be open on VS2013 or revert those lines.